evenOccurences/evenOccurences.js

```diff
@@ -12,4 +12,22 @@
 
 const evenOccurence = (arr) => {
   // Your code here.
+  const obj = {};
+  const occurences = [];
+  arr.forEach((value) => {
+    if (obj[value]) {
+      obj[value]++;
+    } else {
+      obj[value] = 1;
+    }
+  });
+  Object.keys(obj).forEach((key) => {
```

Comment:
This is great. Nice use of Object.keys here. In the past I would've used a for-in loop but I LOVE the .keys/.values methods available now.